### ISSUE TITLE
models attributes: email option doesn't work, instead an email attribute type exists

### ISSUE THREAD

**Issue by stratboy** [balderdashy/sails#3341]
Hi, [docs](http://sailsjs.org/documentation/concepts/models-and-orm/attributes) don't report an 'email' attribute type, but there is and it correctly works.

Instead, docs speak about an 'email' option, so I 'should' use type string and add email:true, but if so, an address like this: 'http://localhost:1337/user/create?username=myusername&email=test@wow.com' won't validate and will throw a validation error.

So, maybe there's something wrong somewhere? :)

**Comment by crobinson42** [comment]
email : {
      type : 'email',
      required : true
    },

**Comment by stratboy** [comment]
Hey there! Thank you but I'm speaking about a documentation problem, not a funcitonal one. I already have working code, I only wanted to point out that the docs are not correct, or at least not complete.

**Comment by sailsbot** [comment]
Thanks for posting, @stratboy.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
